src/detections/detection.rs

```diff
@@ -41,29 +46,59 @@ impl Detection {
                     match quick_xml::de::from_str(&r.data) {
                         Ok(event) => {
                             let event: event::Evtx = event;
-
                             let event_id = event.system.event_id.to_string();
                             let channel = event.system.channel.to_string();
                             let event_data = event.parse_event_data();
 
                             &common.detection(&event.system, &event_data);
                             if channel == "Security" {
-                                &security.detection(
-                                    event_id,
-                                    &event.system,
-                                    &event.user_data,
-                                    event_data,
-                                );
+                                match event_id.as_str() {
+                                    "7030" | "7036" | "7045" | "7040" | "104" => {
+                                        &security.detection(
+                                            event_id,
+                                            &event.system,
+                                            &event.user_data,
+                                            event_data,
+                                        );
+                                    }
+                                    _ => println!("Not Match is Event ID Num:{}", event_id),
+                                }
                             } else if channel == "System" {
-                                &system.detection(event_id, &event.system, event_data);
+                                match event_id.as_str() {
+                                    "4688" | "4672" | "4720" | "4728" | "4732" | "4756"
+                                    | "4625" | "4673" | "4674" | "4648" | "1102" => {
+                                        &system.detection(event_id, &event.system, event_data);
+                                    }
+                                    _ => println!("Not Match is Event ID Num:{}", event_id),
+                                }
                             } else if channel == "Application" {
-                                &application.detection(event_id, &event.system, event_data);
+                                match event_id.as_str() {
+                                    "2" => {
+                                        &application.detection(event_id, &event.system, event_data);
+                                    }
+                                    _ => println!("Not Match is Event ID Num:{}", event_id),
+                                }
                             } else if channel == "Microsoft-Windows-PowerShell/Operational" {
-                                &powershell.detection(event_id, &event.system, event_data);
+                                match event_id.as_str() {
+                                    "8003" | "8004" | "8006" | "8007" => {
```

Comment:
channelはL81でPowerShellとなっていますが、マッチさせているeventIDがAppLockerのものになっています
以下のように変更していただくと正しいと思いますがいかがでしょうか。

```suggestion
                                    "4103" | "4104" => {
```

Reply:
46022cc により修正をしました。